src/main/java/uk/gov/beis/els/categories/lamps/service/LampsService.java

```diff
@@ -141,4 +143,43 @@ public ProcessedEnergyLabelDocument generateHtml(LampsFormPackagingArrow form, L
         .asProcessedEnergyLabelLampsPackagingArrow(ProductMetadata.LAMPS_PACKAGING_ARROW, form);
   }
 
+  public ProcessedEnergyLabelDocument generateHtml(LampsPreSeptember2021ApiForm form,
+                                                   LegislationCategory legislationCategory) {
+    return new TemplatePopulator(templateParserService.parseTemplate("labels/lamps-light-sources/lamps.svg"))
+        .setCondensingMultilineText("supplier", form.getSupplierName())
+        .setCondensingMultilineText("model", form.getModelName()).setRatingArrow("rating",
+            RatingClass.valueOf(form.getEfficiencyRating()), legislationCategory.getPrimaryRatingRange())
+        .setText("kwh", form.getEnergyConsumption())
+        .asProcessedEnergyLabel(ProductMetadata.LAMPS_FULL, form);
+  }
+
+  public ProcessedEnergyLabelDocument generateHtml(LampsPostSeptember2021ApiForm form,
```

Comment:
Sorry should have mentioned this in the Jira, but to avoid having to duplicate these service methods think it would be better to add a method that converts an API form into the normal form. That way the existing template population logic can be used.
e.g. 
```
 public LampsForm toStandardLampsForm(LampsPostSeptember2021ApiForm apiForm) {
    LampsForm form = new LampsForm();
    form.setApplicableLegislation(LEGISLATION_CATEGORY_POST_SEPTEMBER_2021.getId());
    form.setSupplierName(apiForm.getSupplierName());
    form.setModelName(apiForm.getModelName());
    form.setEfficiencyRating(apiForm.getEfficiencyRating());
    form.setTemplateSize(apiForm.getTemplateSize());
   ... etc
  } 

  public LampsForm toStandardLampsForm(LampsPreSeptember2021ApiForm apiForm) {
    LampsForm form = new LampsForm();
    form.setApplicableLegislation(LEGISLATION_CATEGORY_PRE_SEPTEMBER_2021.getId());
    form.setSupplierName(apiForm.getSupplierName());
    form.setModelName(apiForm.getModelName());
    form.setEfficiencyRating(apiForm.getEfficiencyRating());
    ... etc
  }
```
And then call that before passing the regular service methods. Thats the pattern we have used elsewhere e.g. https://github.com/UKGovernmentBEIS/energy-label-service/blob/api-develop/src/main/java/uk/gov/beis/els/categories/waterheaters/service/WaterHeatersService.java#L133